module-apps/application-calllog/ApplicationCallLog.cpp

```diff
@@ -133,9 +133,7 @@ namespace app
         DBServiceAPI::GetQuery(this,
                                db::Interface::Name::Notifications,
                                std::make_unique<db::query::notifications::Clear>(NotificationsRecord::Key::Calls));
-        const auto [succeed, _] = DBServiceAPI::GetQuery(
-            this, db::Interface::Name::Calllog, std::make_unique<db::query::calllog::SetAllRead>());
-        return succeed;
```

Comment:
This changes the logic a bit -> is it consulted?

Reply:
as stated in the PR description 
`ach calllog app opening is sending db::query::calllog::SetAllRead
which marks all calllog database entries as read and send CRUD
notification to application and hence the list view is rebuilt at least
twice for each app opening.
After deeper investigation it seems like dead feature (at least not
covered in GUI) and I am going to disable it.`